### PR TITLE
feat(AnimeTvOnline): add presence

### PR DESCRIPTION
**Pull request by DwifteJB**
## Description 

Created a presence for AnimeTvOnline. https://github.com/PreMiD/Presences/issues/7001

## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/A/AnimeTvOnline/metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "521762564062052393",
+		"name": "Dwifte#0661"
```
Comment (theusaf) [inline websites/A/AnimeTvOnline/metadata.json:5]:
```suggestion
		"name": "Dwifte"
```

File: websites/A/AnimeTvOnline/metadata.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	},
+	"service": "AnimeTvOnline",
+	"description": {
+		"en": "An Italian site to watch anime online."
+	},
+	"url": "animetvonline.org",
```
Comment (theusaf) [inline websites/A/AnimeTvOnline/metadata.json:11]: Site supports `www` subdomain.
```suggestion
	"url": ["www.animetvonline.org", "animetvonline.org"],
```

File: websites/A/AnimeTvOnline/presence.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
+const presence = new Presence({
+	clientId: "1042875166021124229",
+});
+
+enum Assets {
+	logo = "https://i.imgur.com/b0PVWvC.png"
+}
```
Comment (theusaf) [inline websites/A/AnimeTvOnline/presence.ts:7]: Asset names should be capitalized. Also, with so few assets, an enum may not be needed.
```suggestion
enum Assets {
	Logo = "https://i.imgur.com/b0PVWvC.png"
}
```

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/A/AnimeTvOnline/presence.ts (hunk 3 above)
Comment (DwifteJB) [reply websites/A/AnimeTvOnline/presence.ts:7]: Planning on updating this with new features, an enum would be the best move right now.

**Comment by DwifteJB** [comment]
Addressed comments.

**Comment by EncryptedDev** [comment]
Low quality

**Pull request closed without merging**